### PR TITLE
Feature/paint 024

### PR DESCRIPTION
**Pull request by haubitzer**

### CODE REVIEW

File: app/src/main/java/at/tugraz/ist/swe/MainActivity.java [hunk 1]
```diff
@@ -1,111 +1,120 @@
...
+            }
+
+            @Override
+            public void onDismissWithoutSelection(FlyoutMenuView flyoutMenuView) {
+            }
+        });
+    }
+
+    @Override
+    public void onDestroy(){
```
Comment (Yarmek) [inline app/src/main/java/at/tugraz/ist/swe/MainActivity.java]: Don't make dlg_color public and access it directly. Instead implement a dismiss function in the color picker that can be called from the main activity which return the current selected color from the text box.

File: app/src/main/java/at/tugraz/ist/swe/ColorPicker.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
     private EditText textBox_Blue;
     private EditText textBox_Hex;
 
-    private AlertDialog dlg_color;
+    public AlertDialog dlg_color;
```
Comment (Yarmek) [inline app/src/main/java/at/tugraz/ist/swe/ColorPicker.java]: please revert this change back to private

Comment (uwe-h) [reply app/src/main/java/at/tugraz/ist/swe/ColorPicker.java]: Done.

File: app/src/main/java/at/tugraz/ist/swe/MainActivity.java (hunk 1 above)
Comment (uwe-h) [reply app/src/main/java/at/tugraz/ist/swe/MainActivity.java]: Done.
